ui/app/src/components/ContentTypeManagement/controls/TypeDestinationPathsSelector.tsx

```diff
@@ -37,38 +36,27 @@ interface DestinationPaths {
 	excludes: string[];
 }
 
+// TODO: This control should handle xml instead of json.
```

Comment:
This is not very clear. The control shouldn't work with either, it should work with memory data structures (objects). How it is serialised, is what would be best as XML instead of escaped/encoded JSON.